modules/admin_manual/pages/qnap/index.adoc

```diff
@@ -0,0 +1,88 @@
+
+. Create a storage pool in addition to the system volume. You can combine hard disks and solid state disks in RAID groups for flexible storage. This pool can be resized later if necessary.
+
+. In the AppCenter, go to the QNAP Store and install Container Station version 2.0 or higher, which provides virtualization and Docker technogies. 
+
+. Enable `ssh` (secure shell) in the ControlPanel. In the navigation bar on the left side, select menu:Networks & File Services[Telnet / SSH]. Check `Allow SSH connection` and specify the port number (default 22). Next, check `Enable SFTP`. Once you click btn:[Apply], your admin user can log in to your NAS remotely.
+
+== Installation of ownCloud
+
+Go to the AppCenter and select QNAP Store.
+
+Search for ownCloud(_10.8.0_x86_64.qpkg) and click btn:[Install]. Confirm that you want to install this package. During installation you'll see a progress bar displaying the percentage.
+
+After a successful installation, ownCloud 10.8.0 will show up in the AppCenter under menu:QNAP Store[My Apps]. Click on btn:[Open] and proceed with the configuration.
+
+== Configuration
+
+=== First steps
+
+Log in as user `admin`. The default password is `admin` until you change it. In the upper right-hand corner, click on menu:Admin[Settings]. In the left hand navigation bar you find personal configuration options and admin-specific options. Under menu:Personal[General] you can change the password of you admin user.
+
+For more information on the Web interface, refer to the xref:user_manual:webinterface.adoc[Web UI] documentation.
+
+=== Configure your email.
+
+Email is an essential tool in ownCloud for notifying users, user registration, password resets, etc. Logged-in as admin, you can configure email in the menu:Admin[General] settings.
+
+For more information, refer to the section xref:configuration/server/email_configuration.adoc[Email Configuration].
+
+=== Cron
+
+On the same page (menu:Admin[General]) you find the Cron settings.
+
+Make sure Cron remains enabled. This daemon takes care of service jobs in 15-minute intervals.
+
+NOTE: In general, we recommend to keep the default settings in the admin section.
+
+== User Management
+
+== Enabling ownCloud Apps
+
+With the free community edition, you can use all apps that are published under the GPL or similar public licenses, e.g. the GNU Affero General Public License. These you can simply install and enjoy.
+In the upper left corner, click on menu:Files[Market]. Here you can have your pick.
+
+A bundle of Enterprise Apps is published under the ownCloud Commercial License and only available with the Enterprise Edition. To see what's not included in the free community edition, select `App Bundles` in the left-hand navigation bar.
+
+== Enterprise Edition and Licensing
+
+If you need more users than five or want to use enterprise features, obtain an Enterprise License and possibly additonal user packages via the QNAP Store and activate them in the QNAP License Manager.
```

Comment:
I'm not really sure if we should call it an `Enterprise License` at any time. Because we don't have it technically and you cannot activate an `Enterprice License` on QNAP (this also applies to the trial mechanism shown here https://owncloud.com/try-owncloud-enterprise/).
We have QNAP licenses and they activate an Enterprise Edition.

Reply:
Adjusted.